site/docs/reference/Connectors/capture-connectors/mailchimp.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 10
+---
+
+# Mailchimp
+
+This connector captures data from a Mailchimp account.
+
+Three data resources are supported, each of which is mapped to a Flow collection: lists, campaigns, and email activity.
+
+[`ghcr.io/estuary/airbyte-source-mailchimp:dev`](https://ghcr.io/estuary/airbyte-source-mailchimp:dev) provides the latest connector image. You can also follow the link in your browser to see past image versions.
+
+This connector is based on an open-source connector from a third party, with modifications for performance in the Flow system.
+You can find their documentation [here](https://docs.airbyte.com/integrations/sources/mailchimp),
+but keep in mind that the two versions may be significantly different.
+
+## Prerequisites
+
+There are two ways to authenticate with MailChimp when capturing data: using OAuth, and with an API key.
+Their prerequisites differ.
+
+OAuth is recommended for simplicity in the Flow web app;
+the API key method is the only supported method using the command line.
+
+### Prerequisites for OAuth
+
+:::caution Beta
+OAuth implementation is under active development and is coming soon.
+Use the API key method for now.
+:::
+
+* A Mailchimp account
+
+### Prerequisites using an API key
+
+* A Mailchimp account
+
+* A Mailchimp [API key](https://mailchimp.com/developer/marketing/guides/quick-start/#generate-your-api-key)
+
+## Configuration
+
+You configure connectors either in the Flow web app, or by directly editing the catalog spec YAML.
+See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about using connectors. The values and YAML sample below provide configuration details specific to the Mailchimp source connector.
+
+### Properties
+
+#### Endpoint
+
+The following properties reflect the API Key authentication method.
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| `/credentials` | Authentication  | Authentication Type and Details | object |  |
+| `/credentials/auth_type` | Authentication Type | Authentication type. Set to `apikey`.  | string |  |
+| `/credentials/apikey` | API Key | Your Mailchimp API key | string | |
```

Comment:
Should these 3 be marked as Required?

Reply:
I pull these tables directly from the schema of the connector and try not to change them too much. The idea being, ~one day~, it'll all be integrated and the tables in the docs will stay in sync with any updates to the connector itself. That is to say: it looks like the fields aren't marked required in the actual schema and maybe we should add that there? But in the meantime I will definitely change it manually here!